### ISSUE TITLE
Problem when changing month or year

### ISSUE THREAD

**Issue by IronTony** [stephy/CalendarPicker#221]
Hi and thank you for the package.
I just have a question, maybe I am doing something wrong, but I saw that when I tap on the header bar, on month or year, I can select anothe month or another year.
Unfortunately when I do this the month/year change but the calendar days section doesn't update until I press next or previous.

This is the part I am talking about:
![image](https://user-images.githubusercontent.com/3645225/89326548-c3618300-d68a-11ea-8f37-de73f65d14ef.png)

**Comment by peacechen** [comment]
That sounds like a bug.  Are you using the `scrollable` prop?  Please post a code snippet of your `<CalendarStrip>`

**Comment by IronTony** [comment]
Nope, not using `scrollable`. I will post a snippet of code during this weekend.
Thank you

**Comment by JavierLaguna** [comment]
Hi! I have the same issue and im not using `scrollable`

**Comment by MannySAD** [comment]
I'm also facing this issue did you solve it? (not using scrollable either)

**Comment by IronTony** [comment]
did the same thing. no solution actually 😢

**Comment by peacechen** [comment]
Sorry I've not been able to look into this. It will be a while until I have time to debug it. In the meantime, set breakpoints in `node_modules/react-native-calendar-picker/CalendarPicker/index.js` and any other relevant files to determine what may be causing this bug.

**Comment by saaketnarnolia** [comment]
same issue being faced by me

**Comment by daominh** [comment]
I found temporary solution. In `node_modules/react-native-calendar-picker/CalendarPicker/index.js`
add `this.handleOnPressFinisher({year: year, month: month});` inside `handleOnSelectMonthYear()` function.

**Comment by peacechen** [comment]
Thanks for sharing your solution @daominh 
Would you mind submitting a PR?

**Comment by saaketnarnolia** [comment]
this problem started with the version 7.0.0

**Comment by peacechen** [comment]
Thanks for your patience. This has been fixed in 7.0.1
